scripts/cert-update.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+eval `ssh-agent`
+ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+ssh-add /nimbus/id_rsa
```

Comment:
add the actual ssh key that was pulled from a github secret

https://explainshell.com/explain?cmd=ssh-add+%2Fnimbus%2Fid_rsa